orchagent/routeorch.cpp

```diff
@@ -484,7 +556,22 @@ bool RouteOrch::invalidnexthopinNextHopGroup(const NextHopKey &nexthop, uint32_t
                 return parseHandleSaiStatusFailure(handle_status);
             }
         }
-
+        if (nhopgroup->second.nh_member_install_count)
+        {
+            nhopgroup->second.nh_member_install_count--;
+        }
+        
+        if (nhopgroup->second.nh_member_install_count == 0 && nhopgroup->second.eligible_for_default_route_nh_swap && !nhopgroup->second.is_default_route_nh_swap)
+        {
+            if(nexthop.ip_address.isV4())
```

Comment:
if at this time the default route from bgp is not present. will the `v4_active_default_route_nhops` have the drop port?

Reply:
@arlakshm : if no default route than existing behavior will happen where nexthop group will not have any members which will cause drop as expected.